### PR TITLE
Ac 34599 tf provider run on tenant

### PR DESCRIPTION
**Pull request by yuvalstyrbrod**

### CODE REVIEW

File: provider/resource_tenant_role_test.go [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
 
 import (
+	"fmt"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"os"
 	"testing"
 )
 
-const testTenantRole = `
+func testTenantRole(name string) string {
```
Comment (gitomics) [inline provider/resource_tenant_role_test.go:10]: name -> entityId

File: provider/data_source_ssh_client_test.go [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:          func() { testAccPreCheck(t) },
 		ProviderFactories: newTestAccProviders,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccResourceSshClients,
+				Config: testAccResourceSshClients(sshClientName),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(resourceName, "id", "6ddace8e-39a3-4cd3-bba7-ad26e826df5b"),
```
[the remark this answers is not in the document]
Comment (yuvalstyrbrod) [reply provider/data_source_ssh_client_test.go:25]: i working on putting them on circle broadcom

File: Makefile [hunk 3]
```diff
@@ -28,10 +28,46 @@ windows:
 	zip -j release/$(BINARY_NAME)-windows.zip $(OUTPUT_DIR)/windows/*
 
 testacc:
-	export LUMINATE_API_ENDPOINT="${TERRAFORM_ACCEPTANCE_TENANT}" && \
-	export LUMINATE_API_CLIENT_ID="${TERRAFORM_ACCEPTANCE_CLIENT_ID}" && \
-	export LUMINATE_API_CLIENT_SECRET="${TERRAFORM_ACCEPTANCE_CLIENT_SECRET}" && \
-	export TF_ACC=1 && $(GOTEST) -p 1 -v  ./...
+	export LUMINATE_API_ENDPOINT="api.terraformat.luminatesec.com" && \
+	export LUMINATE_API_CLIENT_ID="16a49c4909f4165f7ad33f1d022d83fc" && \
+	export LUMINATE_API_CLIENT_SECRET="734ea9a0d4ae44c55f00773a14538f505d16058fc892286b17a0219be4f84c54" && \
```
Comment (cycode-admin) [inline Makefile:33]: :exclamation:**Cycode**: Secret of type: 'Hex Entropy' was found.
Severity: Medium

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

Comment (cycode-admin) [inline Makefile:33]: :exclamation:**Cycode**: Secret of type: 'Hex Entropy' was found.
Severity: Medium

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

File: Makefile [hunk 4]
```diff
@@ -28,10 +28,46 @@ windows:
 	zip -j release/$(BINARY_NAME)-windows.zip $(OUTPUT_DIR)/windows/*
 
 testacc:
-	export LUMINATE_API_ENDPOINT="${TERRAFORM_ACCEPTANCE_TENANT}" && \
-	export LUMINATE_API_CLIENT_ID="${TERRAFORM_ACCEPTANCE_CLIENT_ID}" && \
-	export LUMINATE_API_CLIENT_SECRET="${TERRAFORM_ACCEPTANCE_CLIENT_SECRET}" && \
-	export TF_ACC=1 && $(GOTEST) -p 1 -v  ./...
+	export LUMINATE_API_ENDPOINT="api.terraformat.luminatesec.com" && \
+	export LUMINATE_API_CLIENT_ID="16a49c4909f4165f7ad33f1d022d83fc" && \
```
Comment (gitomics) [inline Makefile:32]: why adding secret and is here? seems like env vars is safer

**Review by gitomics — APPROVED** [review]

**Pull request merged**